roles/edpm_kernel/defaults/main.yml

```diff
@@ -36,3 +36,4 @@ edpm_nova_compute_config_dir: /var/lib/config-data/ansible-generated/nova_libvir
 
 # KSM control
 edpm_kernel_enable_ksm: false
+edpm_use_bootc: false
```

Comment:
We probably need a better way to implement this globally. But at least for testing purposes, this is what I've used to get something that deploys.

Reply:
This is why I added the edpm_bootc role in https://github.com/openstack-k8s-operators/edpm-ansible/pull/813 so that we had a way to do it consistently across anywhere that needs it.

Reply:
Also, I created a new bootc branch for edpm-ansible: https://github.com/openstack-k8s-operators/edpm-ansible/tree/bootc

Can you propose this PR to the bootc branch instead?

I'll be reverting https://github.com/openstack-k8s-operators/edpm-ansible/pull/813 from main until we are ready to merge all bootc support into main.

Reply:
Branch thing is done. But that role would need to be called from each and every playbook to detect and set the `bootc` variable right? I guess we can just add it as a ansibleVar and avoid calling the role each and every time we start a new service.

Reply:
So, we can use a custom fact. For example:
```
cat /etc/ansible/facts.d/bootc.fact
#!/usr/bin/env bash

is_bootc() {
  BOOTC_STATUS=$(sudo bootc status --json | jq .status.type)
  if [[ "$BOOTC_STATUS" == \"bootcHost\" ]]; then
     BOOTC_SYSTEM="true"
  else
     BOOTC_SYSTEM="false"
  fi
}

is_bootc

echo ${BOOTC_SYSTEM}
```

This is good from the perspective of not needing the user to manually define that they are using a bootc system, plus it works for our non bootc systems:
```
[m3@osp-df-3 bootc]$ ansible -i inv.yaml all -m setup -a "filter=ansible_local"
edpm-compute-1 | SUCCESS => {
    "ansible_facts": {
        "ansible_local": {
            "bootc": true
        },
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false
}
```

So you could combine them in the same NodeSet if you wanted to. But, the down side of this approach is that we need to gather facts from each service. We have thus far tried to limit the amount of fact gathering required, so this approach my not be what we want to do without some more granular control of which facts are being gathered in each service. At the moment, we just define a variable for `gather_facts`. If that variable is true, then we gather all facts. Obviously, that becomes necessary if we want to allow individual task executions that require facts, but when we want to _just_ gather local facts, then gathering all of them introduces non-trivial time to our executions of each service.

Offering it as a potential solution that we can debate. The alternative is that we require either bootc or non-bootc nodes in each NodeSet.